power-outage-monitor/README.md

```diff
@@ -0,0 +1,122 @@
+
+```json
+{
+  "req": "hub.set",
+  "mode": "periodic",
+  "inbound": 1440,
+  "outbound": 60,
+  "product": "com.your-company:your-product-name",
+  "sn": "your-serial-number"
+}
+```
+
+> This app uses `periodic` mode to minimize power use so that the solution can remain online for longer when mains power fails. For more details, see [`hub.set` requests](https://dev.blues.io/notecard/notecard-walkthrough/essential-requests/#notehub-configuration).
+
+### Configure Power Outage Alerts
+
+Paste this command into the in-browser terminal
+
+```json
+{ "req": "card.voltage", "mode": "lipo", "usb": true, "alert": true, "sync": true }
+```
+
+> This instructs the Notecard to assume LiPo battery characteristics, monitor USB power and send an alert immediately when a power outage occurs and when power is restored. For more details, see the [Notecard API reference](https://dev.blues.io/reference/notecard-api/card-requests/#card-voltage) and the `Enable USB Power Alerting` example.
+
+### Notecard CLI
+
+If you want to issue all of the above requests in one shot, you can do so with the [Notecard CLI](https://dev.blues.io/tools-and-sdks/notecard-cli/) and the config.json configuration script. You will need to change the ProductUID in the `hub.set` command to match your ProductUID. Once you've done that, you can configure the Notecard with:
+
+```sh
+notecard -setup config.json
+```
+
+## System Test
+
+In this section, you will power the Notecard via the power brick and simulate a power failure.
+
+### Setup
+
+1. Insert the power brick into a suitable electrical outlet and provide power to the Notecarrier via the micro USB cable.
+
+1. Plug the JST connector from the LiPo battery into the port labeled LIPO on the Notecarrier.
+
+1. Wait for the Notecard to connect to the cloud.  When connected, there will be a new session event in Notehub. (look for recent `_session.qo` events.)
+
+1. Wait for the battery to fully charge.
+
+### Simulating a Power Outage
+
+1. Remove the power brick from the electrical outlet to simulate a power outage. The Notecard is now being powered by the battery.
+
+2. After a short time, you will see a  `_health.qo` event in Notehub with the body
+
+```json
+{"text":"USB power OFF {usb-disabled}"}
+```
+
+This event indicates that the Notecard is no longer powered from the power brick.
+
+3. Reinsert the power brick into a suitable electrical outlet to simulate power being restored.
+
+4. After a short time, you will see a  `_health.qo` event in Notehub with the body
+
+```json
+{"text":"USB power ON {usb-enabled}"}
+```
+
+This event indicates that the Notecard is again powered by the power brick.
+
+
+### Blues Wireless Community
+
+We’d love to hear about you and your project on the [Blues Wireless Community Forum].
```

Comment:
Should we move the link below here? i.e.:

```
[Blues Wireless Community Forum](https://discuss.blues.io/)
```

Reply:
I don't mind either way. I copied it from some existing docs in the repo.

Reply:
I'll leave it up to you. I'll go ahead and give my seal of approval on the overall PR.